dataeng/resources/warehouse-transforms-ci-dbt.sh

```diff
@@ -11,4 +11,17 @@ dbt deps --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --prof
 dbt seed --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
 
 dbt run $DBT_RUN_OPTIONS $DBT_RUN_EXCLUDE --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
-dbt test $DBT_TEST_OPTIONS $DBT_TEST_EXCLUDE --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
\ No newline at end of file
+dbt test $DBT_TEST_OPTIONS $DBT_TEST_EXCLUDE --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET && true
+if [ $? -eq 1 ]
+then
+    if [ "$WITH_RETRY" == "true" ]
+    then
+        pip install -r ../../tools/ci_scripts/requirements.txt
+        NO_OF_TRIES=3
+        python ../../tools/ci_scripts/rerun_flaky_tests.py --project-path . --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/  --profile $DBT_PROFILE  \
+        --target $DBT_TARGET --count $NO_OF_TRIES
```

Comment:
^ I think you will want to include the exclusions in the invocation of this script. The script has an optional flag `--exclusions` that should consume the same exclusions as the original test run